CONTRIBUTING.rst

```diff
@@ -193,3 +199,48 @@ different depending on the context.
 
 Tip! Try to mock in limited context so that individual tests don't affect other
 tests. Use context managers instead of monkey patching imported modules.
+
+
+Updating Your Local Instance
+============================
+When changes are merged to the main Pontoon repository, you'll want to update
+your local development instance to reflect the latest version of the site. You
+can use Git as normal to pull the latest changes, but if the changes add any new
+dependencies or alter the database, you'll want to install any new libraries and
+run any new migrations.
+
+If you're unsure what needs to be run, it's safe to just perform all of these
+steps, as they don't affect your setup if nothing has changed:
+
+.. code-block:: shell
+
+   # Pull latest code (assuming you've already checked out master).
+   git pull origin master
+
+   # Install new dependencies or update existing ones.
+   pip install -U --force --require-hashes -r requirements.txt
+
+   # Run database migrations.
+   python manage.py migrate
```

Comment:
Those instructions will not work for someone using ``docker``. I suggest we move them to the install docs instead, and add an equivalent to the install with docker doc as well.

Reply:
Right. Since the rest of this document assumes the docker environment (see Tests section for example), we could also put docker docs here. On the long term we probably want to document developer setup with docker only.